### ISSUE TITLE
ds.subset does not allow subset by single column

### ISSUE THREAD

**Issue by paulmorio** [datashield/dsBaseClient#13]
Hello,

I just wanted to note that producing a subset of a dataframe with one column returns an error. Our current work-around for this is to create a dummy column using datashield.assign and then doing a subset using two columns. 

Example:
```R
ds.subset(x = 'D', subset = 'test', cols = c('AGE_BASE'))
Error: Command 'subsetDS(dt = "D", complt = FALSE, rs = NULL, cs = "AGE_BASE")' failed on 'InterAct_denmark': Error while evaluating 'base::assign('test', value={dsBase::subsetDS( base::'='( dt,"D" ),base::'='( complt,FALSE ),base::'='( rs,NULL ),base::'='( cs,"AGE_BASE" ) )})'
```

Cheers,
Paul

**Comment by pb51** [comment]
This function is now deprecated by ds.dataFrame.subset  and ds.subset should be removed. This has been discussed and agreed with Stuart (for reasons unrelated to this post). Olly how should we now deal with this - can post be closed and if so by whom?

**Comment by OllyButters** [comment]
Closing issue as long term solution now available (since v5.0) and workaround no longer needed.
